op-e2e/op_geth_test.go

```diff
@@ -140,7 +140,7 @@ func TestGethOnlyPendingBlockIsLatest(t *testing.T) {
 		require.Equal(t, pendingBalance, aliceStartBalance, "pending balance must still be the same")
 	}
 
-	startBlock, err := opGeth.L2Client.BlockByNumber(ctx, nil)
+	// startBlock, err := opGeth.L2Client.BlockByNumber(ctx, nil)
```

Comment:
No need to skip this, we are dealing with a L2 block that will contain the BaseFee.

Reply:
done